src/components/templates/PosterHall/PosterHall.tsx

```diff
@@ -41,8 +60,36 @@ export const PosterHall: React.FC<PosterHallProps> = ({ venue }) => {
     ));
   }, [posterVenues]);
 
+  useEffect(() => {
+    const interval = setInterval(() => {
+      const liveVenueIds = relatedVenueEvents
+        .filter((event) => isEventLive(event))
+        .map((event) => event.venueId);
+      const livePosterRelatedVenues = posterRelatedVenues.filter(
+        (posterRelatedVenue) => liveVenueIds.includes(posterRelatedVenue.id)
+      );
+      setPosterRelatedPreviews(livePosterRelatedVenues);
+    }, 1000);
+    return () => clearInterval(interval);
+  }, [posterRelatedVenues, relatedVenueEvents]);
```

Comment:
> Also, currently having difficulty getting the event to update live (though refreshing does the trick). Any react tips to address that would be appreciated.

@margulies Instead of using a `useEffect` + `setInterval` here, try just using `useInterval`.

The 'interval time' here will need to be extracted to `src/settings` or one of the existing ones defined there should be used.

Reply:
@margulies Though taking a step further back here.. what is the actual intent/desire for this code? There may be a simpler/better way to go about it.

Reply:
Thanks for the feedback — to explain the aim of this PR: We have events related to the posterhall that will only be occuring for the first half of each session. So as not to clutter the main landing page (partymap), and to emphasize the relationship to the posterhall, I would ideally like the venues related to these event to appear at the top of the posterhall for only the half hour in which they are occuring. Happy for any other thoughts related to implementation. Otherwise, I'll continue addressing your comments here. Thanks!

Reply:
I've now implemented with `useInterval` and the additional refactoring you suggested. Thanks!